deeplearning/ml4pl/poj104/dataset_test.py

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env python3
 
+import os
 from deeplearning.ml4pl.poj104 import dataset
```

Comment:
I wouldn't mind if you would rather import `pytest` here, but as is looks perfect too.

Reply:
That would be the "right way" :) at some point I'd like to pull this code back into the main ProGraML repo, will do that then